frontend/src/components/search/Search.tsx

```diff
@@ -30,9 +33,12 @@ const Search: FunctionComponent<SearchProps> = ({
 
   return (
     <div
-      className={`${classes.root} ${className} ${
-        searchDisabled && classes.disabled
-      }`}
+      className={classNames(
+        `${classes.root} ${className} ${searchDisabled && classes.disabled}`,
```

Comment:
Nit: I think you can better leverage the `classnames` library here. Maybe this could be rewritten to:

```js
classNames(classes.root, className, {
  [classes.disabled]: searchDisabled,
  [classes.responsive]: responsive
})
```

?